package-lock.json

```diff
@@ -10,58 +10,58 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
       "dev": true,
       "requires": {
-        "@babel/highlight": "^7.0.0"
+        "@babel/highlight": "7.0.0"
```

Comment:
I think your npm version might be wrong, its removing carats all over the shop 🙈

Reply:
this one is a mystery, I'm using v8.11.3. I'll copy the `package-lock.json` from master tho.